### DISCUSSION TITLE
Please Check This Styling

### DISCUSSION THREAD

**Discussion by BD103** [Repl-it-Coders/Repl.it-2020#7] [Show and tell]
Hello everyone. Can you guys look over this [website](https://barrencooperativeobjectdatabase.bd103.repl.co/)?
I'm trying to redesign the style of the website, because it's not very coordinated. Any thoughts?

@whippingdot @firefish111

**Comment by firefish111** [comment]
Okay, It looks like the certifiate on `repl.co` has been borked, so I cannot see the website linked.
Can you send over the stylesheet so I can take a look at it?

**Reply by BD103** [reply]
Sorry about that. I accidentally deleted the repl. You can find it [here](https://repl.it/@BD103/BarrenCooperativeObjectdatabase).

**Comment by Isaiah-Day** [comment]
It looks great! I feel like the color scheme is a bit to bland, but everything else looks great!

**Reply by BD103** [reply]
Yeah. That's what I've gathered too. I'm going to try to grab some of the colors from the original site, then transfer then to W3CSS.

**Comment by whippingdot** [comment]
Sorry for the straightforwardness, but EWWWWWWW!!!! I hate the purple colors. It looks HORRIBLE. Please no...NOOOO!!! @darkdarcool already made some changes to the CSS and that is enough

**Reply by BD103** [reply]
Ok. Notes to self:
- No purple
- More vibrant
- More ... Chaotic?
- I have no clue what I'm doing honestly

**Reply by whippingdot** [reply]
Not no purple, just don't put EVERYTHING as purple. Also light purple doesn't look good, maybe dark. Also YES, more vibrant. I like light colors in the background personally. Also, can you check the timeline page? I changed it. I added lots of light colors to the website to make it look nice. Check it out [here](https://replit-2021.whippingdot.repl.co/timeline)

**Reply by BD103** [reply]
Ok. So let me get this straight.
- Definitely less purple, more of other colors
- More vibrant and abstract
- Probably a more chaotic font (i think I have an idea)
- Put in some rounded edges and circles. Squares are boring
Does this work?

**Reply by whippingdot** [reply]
Yeah, But put in some circles too, i have a feeling you will wow me.

**Reply by iop3** [reply]
...and this conservation just ends here. excellent!

**Reply by firefish111** [reply]
o
